src/win/fs.c

```diff
@@ -1964,7 +1988,8 @@ void uv_fs_req_cleanup(uv_fs_t* req) {
       uv__free(req->ptr);
   }
 
-  if (req->fs.info.bufs != req->fs.info.bufsml)
+  if (req->fs.info.bufs != req->fs.info.bufsml
+      && ARRAY_SIZE(req->fs.info.bufs) > 0)
```

Comment:
Style nit: The `&&` should be on the previous line.

Reply:
One example of where this convention is used elsewhere: https://github.com/libuv/libuv/blob/v1.x/src/fs-poll.c#L229

Reply:
I went ahead and address this nit, I guess I chose the wrong example to follow.